units/patch_intent.go

```diff
@@ -375,7 +375,7 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 	}
 
 	if patchDoc.IsMergeQueuePatch() {
-		patchDoc.Description = model.MakeCommitQueueDescription(patchDoc.Patches, pref, patchedProject, patchDoc.IsGithubMergePatch(), patchDoc.GithubMergeData)
+		patchDoc.Description = model.MakeCommitQueueDescription(patchDoc.Patches, pref, patchedProject, patchDoc.IsMergeQueuePatch(), patchDoc.GithubMergeData)
```

Comment:
Can we update the name of MakeCommitQueueDescription and remove the "githubMergePatch" bool and the commit queue logic?

Reply:
I moved this function to the job package since it's actually only used here now, and since it's only used for the merge queue case now, I moved it directly to that function rather than this if statement